lib/Group/GroupBackend.php

```diff
@@ -145,4 +146,8 @@ public function usersInGroup($gid, $search = '', $limit = -1, $offset = 0) {
 		return $users;
 	}
 
+	public function getBackendName(): string
+	{
+		return 'WorkspaceGroupBackend';
```

Comment:
Could you replace it with `$this::class`, please?

It must return the real class name. This means choosing between `GroupBackend` and `WorkspaceGroupBackend` in the class name.

Reply:
I made it on purpose because it was an ugly full class name with some \\
If we draw "GroupBackend", this will give no information

Reply:
It's already the natural way:

            "SPACE-U-7": {
                "gid": "SPACE-U-7",
                "displayName": "U-Fifth",
                "types": [
                    "Database",
                    "OCA\\Workspace\\Group\\GroupBackend"
                ]
            }

I made the PR not to have this full class path, that is unpleasant to use or compare. In order to have:

            "SPACE-U-7": {
                "gid": "SPACE-U-7",
                "displayName": "U-Fifth",
                "types": [
                    "Database",
                    "WorkspaceGroupBackend"
                ]
            }

As far as i know, this type is not used in the front part. So that's not very important and doesn't really justify to work on it more than 5 min. If you prefer the backend name by the object class, you may just drop the PR.

Reply:
Good point for the type! Maybe we don't need it :eyes: 

We can delete this I think(?)

But, for more consistency

Reply:
Why ?
eg : /var/www/html/nc29-dev/apps/user_ldap/lib/Group_Proxy.php

Reply:
Okay, I accept your PR